### PR TITLE
Add greedy method for longest palindromic subsequence

### PR DESCRIPTION
**Pull request by The-Kid-Gid**
### **Describe your change:**

Add a recursive backtracking programming solution to the longest palindromic subsequence problem.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by xcodz-dot** [comment]
@dhruvmanila check this
Also, I think it will end up with recursion error at some point with long strings

File: greedy_method/greedy_longest_palindromic_subsequence.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+"""
+The longest palindromic subsequence (LPS) is the longest
+subsequence in a string that is the same when reversed.
+A subsequence is not the same as a substring.
+Unlike substrings, the characters of subsequences are not required
+to occupy consecutive positions.
+
+Explanation:
+https://www.techiedelight.com/longest-palindromic-subsequence-using-dynamic-programming/
```
Comment (dhruvmanila) [inline greedy_method/greedy_longest_palindromic_subsequence.py:9]: This code is a direct copy from the mentioned link.
@cclauss Please take a look at this.

Comment (xcodz-dot) [reply greedy_method/greedy_longest_palindromic_subsequence.py:9]: true, only a few names are changed which was recommended by bot

**Comment by cclauss** [comment]
Closing.  This is code copied from elsewhere and we are already planning to remove the greedy_method because it does not fit in with our other root directories.  #3956

**Comment by The-Kid-Gid** [comment]
@cclauss @dhruvmanila @xcodz-dot  thanks. I followed the explanation of the algorithm from the article and my code turned out very similar. I didn't mean to waste the time of the maintainers, and I understand your decision.

**Pull request closed without merging**